BrickKit/bricks/src/main/java/com/wayfair/brickkit/brick/BrickProvider.kt

```diff
@@ -0,0 +1,24 @@
+package com.wayfair.brickkit.brick
+
+import androidx.annotation.LayoutRes
+
+/**
+ * Provides [BaseBrick]s based on resource ids.
+ *
+ * Copyright © 2017 Wayfair. All rights reserved.
```

Comment:
should the copyright be changed?

Reply:
Usually you do 2017-<this year>

Reply:
https://github.com/wayfair/brickkit-android/issues/130